package-res/resources/web/dataapi/models-svc.js

```diff
@@ -55,7 +55,7 @@ pentaho.pda.SvcHandler.prototype.getSources = function(callback, options) {
             var contextName = options['context'];
             var query = "domainName=&"+((contextName)? 'context='+contextName : "");
             var jsonStr = pentahoPost( url, query, null, 'text/text' );
```

Comment:
@rfellows, I'm afraid it is a bit wider, than expected from this PR. That is not a problem for me to introduce such changes, but it misleads from the focus of the ticket.

@pamval, @mbatchelor, do you agree with Rob (replacement of `pentahoPost`)